packages/cli/test/integration/generators/relation.has-many.integration.js

```diff
@@ -363,19 +311,14 @@ describe('lb4 relation HasMany', /** @this {Mocha.Suite} */ function () {
       },
       {
         relationType: 'hasMany',
-        sourceModel: 'CustomerClass',
-        destinationModel: 'OrderClass',
-        registerInclusionResolver: true,
```

Comment:
So even `registerInclusionResolver` is different, the test is still a redundant?

Reply:
`true` is the default behaviour. Therefore I think it can be removed, though I don't have strong opinions on it.

Reply:
Thank you for explanation. Hmm, is it related to the decreased test coverage?
There is a -0.03 decrease and the missed lines are marked in: https://coveralls.io/builds/33584789/source?filename=packages/cli/generators/relation/utils.generator.js, L81-L86.

Reply:
@jannyHou thank you for review the PR! I reverted one test and now the coverage drops -0.009%. Do you think it's acceptable?

Reply:
It's much better!